reporthandling/apis/cloudmetadata.go

```diff
@@ -0,0 +1,41 @@
+package apis
+
+import "strings"
+
+type CloudProviderName string
+
+const (
+	GCP CloudProviderName = "GCP" // TODO: Why do we have this name ??
```

Comment:
I left the `GCP` since I copied it from the `report/v2` pkg and I don't know what packages are using it.